providers/pinterest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "pinterest",
+  "services": [
+    {
+      "id": "api",
+      "baseUrl": "https://api.pinterest.com"
+    },
+    {
```

Comment:
what's this service for?

Reply:
(the `www` one; github's inline view is weird)

Reply:
For `https://www.pinterest.com/oauth/`: https://developers.pinterest.com/docs/api/v5/#section/Start-the-OAuth-flow